modules/v1/workspaces/models/TimelinePost.php

```diff
@@ -61,6 +62,10 @@ public function behaviors()
         return array_merge(parent::behaviors(), [
             TimestampBehavior::class,
             BlameableBehavior::class,
+            [
+                'class' => TimelineUrlBehaviour::class,
+                'timelinePost' => $this
```

Comment:
In this specific case there is no need to create behavior, you can even write necessary code in model, but if you want to create universal behavior, better will be if you pass property name(s) as an array, but not $this.

Reply:
This behaviour was created specifically for timeline posts. If same kind of behaviour is needed for other models, i will look into it